core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -33,8 +33,9 @@ public enum PropertyKey implements ConfigPropertyKey {
     BINARYDATA_ENCODING_MAXLENGTH("binarydata.encoding.maxlength"),
     CAIRO_ROOT("cairo.root"),
     CAIRO_VOLUMES("cairo.volumes"),
-    CAIRO_SNAPSHOT_INSTANCE_ID("cairo.snapshot.instance.id"),
+    CAIRO_LEGACY_SNAPSHOT_INSTANCE_ID("cairo.snapshot.instance.id"),
     CAIRO_SNAPSHOT_RECOVERY_ENABLED("cairo.snapshot.recovery.enabled"),
```

Comment:
nit: might be a good idea to add legacy to the enum name: `CAIRO_LEGACY_SNAPSHOT_RECOVERY_ENABLED`

Reply:
done https://github.com/questdb/questdb/pull/4821/commits/54be271306f73d2a04f6caf59638838b561fbaa1